nh_clinical/patient.py

```diff
@@ -84,6 +84,16 @@ def _get_name(self, cr, uid, ids, fn, args, context=None):
             result[r['id']] = self._get_fullname(r)
         return result
 
+    def name_get(self, cr, uid, ids, context=None):
```

Comment:
It's an Odoo convention https://github.com/valdecdev/odoo/blob/openeobs-8-12/openerp/models.py#L1650